src/modules/physics/physics.c

```diff
@@ -53,6 +56,27 @@ void lovrPhysicsDestroy() {
   initialized = false;
 }
 
+void onErrorMessage(int num, const char *msg, va_list args)
+{
+  char composed_msg[1024];
+  vsnprintf (composed_msg, 1024, msg, args);
+  lovrLog(LOG_DEBUG, "ODE", composed_msg);
+}
+
+void onDebugMessage(int num, const char *msg, va_list args)
+{
+  char composed_msg[1024];
+  vsnprintf (composed_msg, 1024, msg, args);
+  lovrLog(LOG_DEBUG, "ODE", composed_msg);
+}
+
+void onInfoMessage(int num, const char *msg, va_list args)
+{
+  char composed_msg[1024];
+  vsnprintf (composed_msg, 1024, msg, args);
+  lovrLog(LOG_INFO, "ODE", composed_msg);
+}
+
```

Comment:
Style notes:

- Opening braces for the functions should be on the same line, like `) {`
- Prefer full words for variables/parameters like `message`
- No spaces after function invocation `vsnprintf(`.
- Use camel case instead of snake case for variables e.g. `composedMessage`
- Pointer stars should be near the type instead of the variable `const char* message`